example/README.md

```diff
@@ -37,9 +37,15 @@ sentry:
   # default 'warning'
   log_level: error
 
-  # default to build/web
+  # default 'build'
+  #build_path: ...
+
+  # default 'web'
```

Comment:
Should we also mention here that this is relative to `build_path`?

Reply:
done